tools/pipelines/templates/build-npm-package.yml

```diff
@@ -32,6 +32,11 @@ parameters:
   default:
   - ci:test
 
+- name: testResultDirs
```

Comment:
Can we document the expected format here, since it's typed as an arbitrary object.

Reply:
It would probably be best to document the semantics as well.

Reply:
Added comment. Thanks!